EventPlanning.DAL/Event.cs

```diff
@@ -15,7 +15,7 @@ public class Event
         public DateTime DateFrom { get; set; }
         public DateTime DateTo { get; set; }
         public string Address { get; set; }
-        public List<EventActivity> Activities { get; set; }
+        public virtual ICollection<EventActivity> EventActivities { get; set; }
```

Comment:
Is it `ICollection`? Maybe `IEnumerable` would be the best choice? What is the difference?